app/networkapi/templates/highlights/adminsortable_objects_custom.html

```diff
@@ -0,0 +1,19 @@
+{% load adminsortable_tags admin_urls staticfiles %}
```

Comment:
This is a copy/paste job from "people". I want to put this somewhere common, but I'm not sure I can.